py_gtfs_rt_ingestion/tests/test_convert.py

```diff
@@ -1,9 +1,37 @@
 import os
+import pyarrow
+from unittest.mock import patch
+from pyarrow import fs
 
-from py_gtfs_rt_ingestion import Configuration, gz_to_pyarrow
+from py_gtfs_rt_ingestion import Configuration
+from py_gtfs_rt_ingestion import gz_to_pyarrow
+from py_gtfs_rt_ingestion import ConfigType
 
 TEST_FILE_DIR = os.path.join(os.path.dirname(__file__), "test_files")
 
+def test_bad_conversion():
+    bad_return = gz_to_pyarrow(filename='badfile',config=None)
+    assert bad_return == 'badfile'
+    assert not isinstance(bad_return, pyarrow.Table)
+
+    with patch('pyarrow.fs.S3FileSystem', return_value=fs.LocalFileSystem):
+        bad_return = gz_to_pyarrow(filename='s3://badfile',config=None)
+    assert  bad_return == 'badfile'
+    assert not isinstance(bad_return, pyarrow.Table)
+
+def test_empty_files(tmpdir):
+    config = Configuration(config_type=ConfigType.RT_VEHICLE_POSITIONS)
+
+    empty_file = os.path.join(TEST_FILE_DIR, "empty.json.gz")
+    table = gz_to_pyarrow(filename=empty_file, config=config)
+    np_df = table.to_pandas()
+    assert np_df.shape == (0,len(config.export_schema))
+
+    one_blank_file = os.path.join(TEST_FILE_DIR, "one_blank_record.json.gz")
+    table = gz_to_pyarrow(filename=one_blank_file, config=config)
+    np_df = table.to_pandas()
+    assert np_df.shape == (1,len(config.export_schema))
+
```

Comment:
Add testing for dummy json.gz files with 0 records and 1 record of empty data. Hopefully covers edge cases...